lib/kafka_listener.rb

```diff
@@ -21,12 +21,12 @@ def subscribe
         raw_process(event)
       end
     end
-  rescue Kafka::ConnectionError => e
-    Rails.logger.error("Cannot connect to Kafka cluster #{messaging_client_options[:host]}")
-    unless messaging_client_options[:host] == 'localhost'
-      sleep 30 # Remote Kafka may be down. Try again later
-      retry
-    end
+  # rescue Kafka::ConnectionError => e
+  #   Rails.logger.error("Cannot connect to Kafka cluster #{messaging_client_options[:host]}")
+  #   unless messaging_client_options[:host] == 'localhost'
+  #     sleep 30 # Remote Kafka may be down. Try again later
```

Comment:
Is there a question this change may not work?

Reply:
I'm not sure, Bill just asked for it removed here: https://github.com/RedHatInsights/catalog-api/pull/815#issuecomment-693651136

What do you think @bzwei ?

Reply:
I am struggling how to detect connection error. We will figure out later.